eng/pipelines/templates/jobs/ci.yml

```diff
@@ -416,20 +416,15 @@ jobs:
       # as we require the GitHub service connection to be loaded.
       - ${{ if not(contains(variables['Build.DefinitionName'], 'java-pr')) }}:
         - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
-          parameters:
-            Paths:
```

Comment:
Removing these paths appears to cause build breaks in the `Compliance` job: 

* Failures -- https://dev.azure.com/azure-sdk/internal/_build/results?buildId=2928450&view=results
* Without failures yesterday -- https://dev.azure.com/azure-sdk/internal/_build/results?buildId=2924626&view=logs&s=6884a131-87da-5381-61f3-d7acc3b91d76&j=bc67675d-56bf-581f-e0a2-208848ba68ca